src/assets/Generator.Shared/NextLinkOperationImplementation.cs

```diff
@@ -26,6 +26,7 @@ internal class NextLinkOperationImplementation : IOperation
 
         private string? _lastKnownLocation;
         private string _nextRequestUri;
+        private string? _apiVersion;
```

Comment:
I'm wondering how this approach will scale to accommodate all of the customizations that could happen in code generated from a swagger file.  For example, there is a recent customer request to handle a 207 response as a request as well - would we handle that here, too?  Or is there a way to add an abstract method (or something similar) to call out to code generated for the library to create the message for polling requests?